### DISCUSSION TITLE
how to pass a custome filter to Azure AI search?

### DISCUSSION THREAD

**Discussion by soppy2024** [microsoft/sample-app-aoai-chatGPT#953] [Q&A]
could someone give me some guidance on how to pass custom filter to Azure AI search query please.

**Comment by stbere** [comment]
Try checking here: https://learn.microsoft.com/en-us/azure/search/search-filters

**Reply by Alongejm** [reply]
I am trying to figure out the same thing, I am just not sure what to modify in the code. I want to add an environment variable to set the filter then modify the code to check if it is empty and if not add it on to the url for the API call
